topostats/tracing/nodestats.py

```diff
@@ -1053,7 +1008,7 @@ def gaussian(x: npt.NDArray, h: float, mean: float, sigma: float):
 
     def fwhm2(self, heights: npt.NDArray, distances: npt.NDArray, hm: float | None = None) -> tuple:
         """
-        Caculate the FWHM value.
+        Calculate the FWHM value. TODO: Dictionary-ify this one to help saving.
```

Comment:
Recommend capturing this in an issue so we can keep track of it (if not already done).

Reply:
That's a remnant I accidentally left in from Max's notes. removing it.